content/100-getting-started/02-setup-prisma/02-start-from-scratch.mdx

```diff
@@ -278,6 +278,8 @@ This command does two things:
 1. It creates a new SQL migration file for this migration
 1. It runs the SQL migration file against the database
 
+> Note: `generate` is called when you run a migration, if the `prisma-client-js` generator is defined in your schema this will check if `@prisma/client` is installed and install it if it's missing.
```

Comment:
```suggestion
> **Note**: `generate` is called under the hood when you run a migration. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
```